### DISCUSSION TITLE
Change chips input border color

### DISCUSSION THREAD

**Discussion by pparage** [mdbootstrap/TW-Elements#1975] [Support from community]
`   <div data-te-chips-input-init data-te-chips-placeholder data-te-label-text="Project Members"
                    class="mb-0 min-h-[45px] pb-0 shadow-none transition-all duration-300 ease-[cubic-bezier(0.25,0.1,0.25,1)] hover:cursor-text"
                    data-te-editable="true"></div>`
                    
The default border color is greyish I want it to be white. I really tried some of the things here https://tailwind-elements.com/docs/standard/components/chips/#docsTabsAPI. But I am simply not able to change the border color.

Thank you in advance.

**Comment by juujisai** [comment]
Hi! You should be able to change the styles with use of the `inputClasses` option, but I see there is some issue with it. I'll add this to our list to check it out

**Comment by Trochonovitz** [comment]
As @juujisai mentioned, everything with component is wokring fine. All you got to do is include your main `.js` file into `tailwind.config.cjs` in `content` object. We are going to update our "Getting started" section soon. Case closed!
